java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/LoggerI.java

```diff
@@ -95,6 +95,11 @@ public Logger cloneWithPrefix(String prefix) {
         return new LoggerI(prefix);
     }
 
+    @Override
+    public void close() throws Exception {
```

Comment:
No need to have the `throws` just because the parent method did:
```suggestion
    public void close() {
```